Corona-Warn-App/src/main/java/de/rki/coronawarnapp/worker/DiagnosisTestResultRetrievalPeriodicWorker.kt

```diff
@@ -89,13 +89,18 @@ class DiagnosisTestResultRetrievalPeriodicWorker(
      * @see TestResult
      */
     private fun initiateNotification(testResult: TestResult) {
+        if (LocalData.isTestResultNotificationSent() || LocalData.submissionWasSuccessful()) {
```

Comment:
return if ...

Reply:
afaik this is not possible as return if always requires an else branch